### ISSUE TITLE
paccheck: "mtree data not available" inconsistency

### ISSUE THREAD

**Issue by mkoskar** [andrewgregory/pacutils#18]
```
$ paccheck --version
paccheck v0.4.0+21-g63df710 - libalpm v10.0.1 - pacutils v0.4.0+21-g63df710
```
When running `pacccheck` without target(s) specified or with all packages on stdin (`< <(pacman -Qq)`) — just for test — I observe a lot of packages to be reported with `mtree data not available`, but when specifying such package directly with the same arguments nothing is reported. Why?

```
$ paccheck --quiet --files --file-properties --sha256sum --backup --noextract --noupgrade
$ paccheck --quiet --files --file-properties --sha256sum --backup --noextract --noupgrade \
    < <(pacman -Qq)
...
zsh: mtree data not available
...
```
but
```
$ paccheck --quiet --files --file-properties --sha256sum --backup --noextract --noupgrade \
    zsh
(no output)
```

**Comment by andrewgregory** [comment]
I can't reproduce this.  Either way I get the warning message (twice, in fact).  Are you sure that you're calling paccheck correctly?  Your --version output indicates the most recent version, but the error output you show looks like it's from an older version.

**Comment by mkoskar** [comment]
You are correct, I've mixed it up while upgrading to -git version, sorry about that :-/. In older version (last release) there is `zsh: mtree data not available` in `stdout` once. In -git version it is `warning: zsh: mtree data not available` twice and it goes to `stderr`. Just for completeness, I've picked up `zsh` as an example, there are lot more packages reported, but not all (just by skimming). So why does it happen? Is it a bug or I'm misunderstanding something.

To summarize:
1) why is there no warning when calling `paccheck` with `zsh` explicitly
2) why is there warning at all? (`/var/lib/pacman/local/zsh-5.3.1-1/mtree` exists)

**Comment by andrewgregory** [comment]
My best guess is that you have so many packages or so little memory that a leak is exhausting your memory or file descriptors so that the file can't be opened.  See if the issue-18 branch fixes the problem.

**Comment by mkoskar** [comment]
I've tested `issue-18` branch, and indeed warnings are gone.

**Comment by andrewgregory** [comment]
Pushed to master; please verify it still works correctly.

**Comment by mkoskar** [comment]
Verified, works ok. Thanks!
